src/base-apis.js

```diff
@@ -1717,22 +1739,49 @@ MatrixBaseApis.prototype.getKeyChanges = function(oldToken, newToken) {
  * @param {string} nextLink Optional If specified, the client will be redirected
  *                 to this link after validation.
  * @param {module:client.callback} callback Optional.
+ * @param {string} isAccessToken The `access_token` field of the Identity Server
```

Comment:
like the react-sdk: `accessToken` or `identityAccessToken` please. Booleans are not strings.

Reply:
Right, sorry for the naming... All fixed! Still not sure why it sounded okay at the time... 😅